plugin/clone/src/clone_os.cc

```diff
@@ -267,8 +274,16 @@ int clone_os_copy_buf_to_file(uchar *from_buffer, Ha_clone_file to_file,
   CLONE_OS_CHECK_FILE(to_file);
 
   while (length > 0) {
+    const auto last_block_used = length % CLONE_OS_ALIGN;
+    const auto padding_size =
+        to_file.o_direct_uneven_file_size
+            ? (CLONE_OS_ALIGN - last_block_used) % CLONE_OS_ALIGN
+            : 0;
+    if (to_file.o_direct_uneven_file_size) {
+      memset(from_buffer + length + 1, 0, padding_size);
```

Comment:
This `+ 1` looks like a literal off-by-one error, leaving one uninitialized byte. Found by Valgrind.

Reply:
Fixed